README.md

```
@@ -13,75 +14,50 @@
+* `BubbleBackgroundView` : 배경 그래픽 구성 (버블 + Gradient)
 
+---
 
 ## 📱 앱 미리보기
 
-| Video | Launcher | Main | Detail |
-|----------|------------|--------------|-------------|
-| <video controls src="docs/assets/divydot.mp4" title="App Demo"></video> | ![](docs/assets/launcher-image.png) | ![](docs/assets/22.54.24.png) | ![](docs/assets/22.54.25.png) |
+| 전체 Progress | 개별 Cycle |
+|----------|------------|
+|![](docs/assets/1.png)|![](docs/assets/2.png)|
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)